packages/components/link/__tests__/link.test.tsx

```diff
@@ -51,17 +51,6 @@ describe("Link", () => {
     expect(onPress).toHaveBeenCalled();
   });
 
-  it("should trigger onClick function", async () => {
-    const onClick = jest.fn();
-    const {getByRole} = render(<Link onClick={onClick} />);
-
-    const link = getByRole("link");
-
-    await user.click(link);
-
-    expect(onClick).toHaveBeenCalled();
-  });
-
   it('should have target="_blank" and rel="noopener noreferrer" when "isExternal" is true', () => {
```

Comment:
@wingkwong do not delete this test until we completely remove the `onClick` support